requirements/base.txt

```diff
@@ -36,7 +36,7 @@ django-autoslug==1.8.0
 pytz==2012h
 
 # background process management
-# django-celery==3.1.16
+django-celery==3.1.16
```

Comment:
Yes, apologies for old docs

`DJANGO_SETTINGS_MODULE=config.settings.dev celery -A seed worker -l info -c 4 --maxtasksperchild=1000 --events`